react/features/video-quality/components/VideoQualityDialog.web.js

```diff
@@ -1,31 +1,31 @@
-import React, { Component } from 'react';
-
-import { Dialog } from '../../base/dialog';
-
-import VideoQualitySlider from './VideoQualitySlider';
-
-/**
- * Implements a React {@link Component} which displays the component
- * {@code VideoQualitySlider} in a dialog.
- *
- * @extends Component
- */
-export default class VideoQualityDialog extends Component {
-    /**
-     * Implements React's {@link Component#render()}.
-     *
-     * @inheritdoc
-     * @returns {ReactElement}
-     */
-    render() {
-        return (
-            <Dialog
-                hideCancelButton = { true }
-                okKey = 'dialog.done'
-                titleKey = 'videoStatus.callQuality'
-                width = 'small'>
-                <VideoQualitySlider />
-            </Dialog>
-        );
-    }
-}
+import React, { Component } from 'react';
+
+import { Dialog } from '../../base/dialog';
+
+import VideoQualitySlider from './VideoQualitySlider';
+
+/**
+ * Implements a React {@link Component} which displays the component
+ * {@code VideoQualitySlider} in a dialog.
+ *
+ * @extends Component
+ */
+export default class VideoQualityDialog extends Component {
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        return (
+            <Dialog
+                hideCancelButton = { true }
+                okKey = 'dialog.done'
+                titleKey = 'videoStatus.callQuality'
+                width = 'small'>
+                <VideoQualitySlider />
+            </Dialog>
+        );
+    }
+}

```

Comment:
This file shows a bunch of changes because the linebreaks have been all changed to CRLF. This is something that needs to be fixed. This error would also be displayed in the linter.

For the linting, can you try removing your node_modules folder and installing again? I can't think of what else is wrong given the limited information I have. What type of failures is it reporting?